### ISSUE TITLE
Version 6.0.5 has broken SQL formatting

### ISSUE THREAD

**Issue by mrcalgary** [dbeaver/dbeaver#6056]
#### System information: 
- Windows 10
- DBeaver 6.0.5
- Java embedded JRE
- Additional extensions

#### Connection specification:
- DB2 11.1.2.2 Linux
- Driver name - DB2 LUV db2jcc4.jar
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:
SQL formatting doesn't work in 6.0.5, worked in the previous versions.
#### Steps to reproduce, if exist:
SELECT sp.*  ,ap.expressionassay_id  ,ap.anatomy_item_id  ,ap.anatomy_item  ,CASE      WHEN ap.biomedia_id IS NOT NULL    THEN RTRIM(cast(sp.expressionstagestested_id AS CHAR(15))) || RTRIM(cast(ap.biomedia_id AS CHAR(15)))    ELSE RTRIM(cast(sp.expressionstagestested_id AS CHAR(15)))  END AS image_pattern_id FROM stage_pattern_v sp     LEFT JOIN
   anatomy_pattern_v ap 
    ON sp.biomedia_id = ap.biomedia_id AND sp.expressionstagestested_id = ap.expressionstagestested_id

Doesn't format nor by CTRL+SHIFT+F nor by right click->format SQL

**Comment by tchibatron** [comment]
What kind of formatter do you use? (Window->Preferences->Database->Editors->SQL Editor->SQL Formatting)
I can not reproduce it yet.

**Comment by mrcalgary** [comment]
It seems to be connected to "Global settings" in "SQL formatting" in "Preferences". If I select the specific datasource formatting settings then for this datasource formatting works. When I deselect datasource specific box, switching back to "Global settings", formatting cease to work. A click on "Global settings" link doesn't do anything. I have done several upgrades on DBeaver from previous versions, and I have played with external formatters. May be in old version I have switched to external formatter in "Global settings", now it's unavailable and that's why I can't switch to "Global settings"? 
So to me formatter works with the connection specific settings but it doesn't work with "Global settings" and I can't modify "Global settings".
